docker-compose/keycloak/Dockerfile

```diff
@@ -10,19 +10,39 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
 
-FROM jboss/keycloak:16.1.1
+#
+# Use builder to integrate custom provider
+#
+FROM quay.io/keycloak/keycloak:20.0.0 as builder
 
-COPY --chown=jboss themes/. /opt/jboss/keycloak/themes
+COPY shanoir-ng-keycloak-auth.jar /opt/keycloak/providers
 
-COPY shanoir-ng-keycloak-auth.jar /opt/jboss/keycloak/standalone/deployments/
+# Configure a database vendor
+ENV KC_DB=mysql
 
-COPY cfg/. /opt/jboss/keycloak/
+# Enable health and metrics support
+ENV KC_HEALTH_ENABLED=true
+ENV KC_METRICS_ENABLED=true
```

Comment:
Are the `/health` and `/metrics` endpoints actually used by shanoir ? If not i would rather let them disabled and avoid exposing any information (if any sysadmin wants them, he can set the env var and put an adequate filter in his reverse proxy so as not to expose them to the public).

Reply:
well done, that you removed it